src/main/java/it/eng/idsa/businesslogic/service/impl/TransportCertsManager.java

```diff
@@ -67,7 +67,7 @@ private void populateTransportCertsSha() {
 			}
 		});
 		transportCerts.remove("localhost");
-		logger.info("Calculated {} transportCertsSha256 values", transportCerts.entrySet().size());
+		logger.info("Calculated transportCertsSha256 values");
```

Comment:
Not needed this log line, now when we do not print transportCerts

Reply:
Removed